zulipterminal/ui.py

```diff
@@ -114,7 +114,11 @@ def get_random_help(self) -> List[Any]:
         return [
             "Help[?] ",
             ("footer_contrast", f" {random_command_display_keys} "),
-            f" {random_command['help_text']}",
+            f" ({tip_context}) ",
+            (
+                "footer_contrast",
+                f" {random_command['help_text']} ",
+            ),
```

Comment:
This shows the information quite well at this stage :+1:

Later, I can see we'll want to iterate on the layout - there's a similar issue as with the `Help [?]` point, but also that the context/keys/descriptions 'bounce around' spacing-wise.